.circleci/config.yml

```diff
@@ -191,80 +187,18 @@ jobs:
           PYTHON_VERSION: py27
           PERCY_ENABLE: 0
 
-  test-legacy-37: &test-legacy
-    working_directory: ~/project
-    docker:
-      - image: circleci/python:3.7.9-stretch-node-browsers
-        auth:
-          username: dashautomation
-          password: $DASH_PAT_DOCKERHUB
-        environment:
-            PYTHON_VERSION: py37
-            PERCY_PARALLEL_TOTAL: -1
-    parallelism: 2
-    steps:
-      - checkout
-      - run: echo $PYTHON_VERSION > ver.txt
-      - restore_cache:
-          key: dep-{{ checksum ".circleci/config.yml" }}-{{ checksum "ver.txt" }}-{{ checksum "dev-requirements.txt" }}
-      - attach_workspace:
-          at: ~/project
-      - run:
-          name: 🧪 Run Legacy Integration Tests
-          command: |
-            . venv/bin/activate && rm -rf dash_core_components && ls -la
-            set -eo pipefail
-            cd packages && mv dash-*.tar.gz main.tar.gz && ls -la
-            find . -name "dash_*.gz" | xargs pip install -I --progress-bar off --no-cache-dir
-            pip install --no-cache-dir --progress-bar off main.tar.gz[dev,testing]
-            pip list | grep dash | xargs pip show && cd ..
-            echo $(python -V 2>&1) | grep 3. && python -m unittest tests/test_dash_import.py
```

Comment:
Only test_dash_import remains and is moved here https://github.com/plotly/dash-core-components/pull/939/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R158